tox.ini

```diff
@@ -1,31 +1,20 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,pypy,pypy3
+   py27,py34,py35,py36,pypy,pypy3,coverage
```

Comment:
Strange indentation change from 4 to 3 spaces.

Reply:
Once again, no idea what happened there. I'll fix. Thanks!